docs/release-notes/february-2024.md

```diff
@@ -0,0 +1,181 @@
+---
+slug: february-2024
+hide_title: true
+description: >-
+    Release notes include the changes, fixes, and additions in specific versions of Semgrep.
+toc_max_heading_level: 3
+title: February 2024
+tags:
+  - Release notes
+---
+
+# Semgrep release notes for February 2024
+
+## 🔧 OSS Engine
+
+* The following versions of the OSS Engine were released in February 2024:
+  * [<i class="fas fa-external-link fa-xs"></i>1.59.1](https://github.com/semgrep/semgrep/releases/tag/v1.59.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.60.0](https://github.com/semgrep/semgrep/releases/tag/v1.60.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.60.1](https://github.com/semgrep/semgrep/releases/tag/v1.60.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.61.1](https://github.com/semgrep/semgrep/releases/tag/v1.61.1)
+  * [<i class="fas fa-external-link fa-xs"></i>1.62.0](https://github.com/semgrep/semgrep/releases/tag/v1.62.0)
+  * [<i class="fas fa-external-link fa-xs"></i>1.63.0](https://github.com/semgrep/semgrep/releases/tag/v1.63.0)
+
+## 🌐 Cloud Platform
+
+### Added
+
+- API: Added a `rule` object under `findings` with the following fields: 
+    - `name`
+    - `message`
+    - `confidence`
+    - `category`
+    - `subcategories`
+    - `technologies`
+    - `vulnerability_classes`
+    - `cwe_names`
+    - `owasp_names` <!-- 12868 --> 
+- Added distinction between Pro engine and OSS findings in the Playground and Editor. <!-- 12275 -->
```

Comment:
@s-santillan @milanwilliams We should remove "Pro engine" and think of better copy to indicate it's Semgrep Commercial. I also wonder if we just skip this for the release notes.

Reply:
I left a similar clarifying question in Slack but posting here too just in case: 

Are we still referring to our commercial offerings as "Semgrep Pro Engine" and "Pro rules" in the app, marketing page, and registry?

Reply:
Resolved with:

> Let’s keep it the same for now and consistent with the app.

No need for any changes.